amical/tests/test_extraction.py

```diff
@@ -22,21 +45,76 @@ def test_add_infos_simulated():
     assert infos["date-obs"] == hdr["DATE-OBS"]
     assert infos["telescop"] == hdr["TELESCOP"]
     assert "observer" not in infos  # Keys that are not in hdr should not be in infos
+    assert "observer" not in infos.hdr  # Keys that are not in hdr should still not be
```

Comment:
looks like the comment was truncated ?

Reply:
Indeed, I will clarify this in a new commit.